File: 백준/Bronze/10807. 개수 세기/개수 세기.c
#include<stdio.h>
int main(){
    int N, array[100], v;
    int cnt = 0;
    
    scanf("%d", &N);
    for (int i = 0; i < N; i++) {
        scanf("%d",&array[i]);
    }

    scanf("%d", &v);
    for (int j = 0; j < N; j++) {
        if(array[j] == v)
            cnt++;
    }
    printf("%d", cnt);
    return 0;
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int anArray[] = new int[100];
        int N = sc.nextInt();
        int cnt = 0;
        
        for (int i = 0; i < N; i++) {
            anArray[i] = sc.nextInt();
        }
        
        int v = sc.nextInt();
        for (int j = 0; j < N; j++) {
            if(anArray[j] == v)
                cnt++;
        }
        System.out.println(cnt);
        
        sc.close();
    }
}